### PR TITLE
fix(jobs): Bump kafka setup resources + make all jobs resources configurable

### PR DESCRIPTION
**Pull request by pedro93**
Fixes https://github.com/acryldata/datahub-helm/issues/221

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: charts/datahub/templates/elasticsearch-setup-job.yml [hunk 1]
```diff
@@ -90,12 +90,7 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           resources:
-            limits:
-              cpu: 500m
```
Comment (szalai1) [inline charts/datahub/templates/elasticsearch-setup-job.yml:94]: Could you make these parameters: like here: https://github.com/acryldata/datahub-helm/pull/223

File: charts/datahub/templates/kafka-setup-job.yml [hunk 2]
```diff
@@ -100,12 +100,7 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           resources:
-            limits:
```
Comment (szalai1) [inline charts/datahub/templates/kafka-setup-job.yml:103]: here too

File: charts/datahub/templates/postgresql-setup-job.yml [hunk 3]
```diff
@@ -79,12 +79,7 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           resources:
-            limits:
```
Comment (szalai1) [inline charts/datahub/templates/postgresql-setup-job.yml:82]: same

File: charts/datahub/templates/elasticsearch-setup-job.yml (hunk 1 above)
Comment (pedro93) [reply charts/datahub/templates/elasticsearch-setup-job.yml:94]: Would that be better than asking a user to define the entire `resource` block? What is the advantage here?
The block version is more verbose but also grants the most flexibility.

**Review by szalai1 — APPROVED** [review]

**Review by david-leifker — APPROVED** [review]

**Comment by adamhowardprice** [comment]
Thank you! Can you deploy version 0.2.123 so I can use this?

**Comment by adamhowardprice** [comment]
@david-leifker

File: .github/workflows/release.yaml [hunk 4]
```diff
@@ -1,47 +0,0 @@
-
-
-name: Release Charts
```
Comment (adamhowardprice) [inline .github/workflows/release.yaml:3]: why did you remove this? I could use the deployed version of master to unblock myself with https://github.com/acryldata/datahub-helm/issues/221

Comment (david-leifker) [reply .github/workflows/release.yaml:3]: This was removed temporarily until this fix is validated in a staging environment. I believe this would be restored when tested. I assume there wasn't an easy way to temporarily disable this.
